task/host-sp-comms/src/bsp/cosmo_a.rs

```diff
@@ -203,46 +206,52 @@ impl ServerImpl {
                     voltage_sensors: SensorId::into_u32_array(sensors.voltage),
                     current_sensors: SensorId::into_u32_array(sensors.current),
                 };
-                self.tx_buf.try_encode_inventory(sequence, &name, || {
-                    use pmbus::commands::isl68224::CommandCode;
-                    let InventoryData::Isl68224 {
-                        mfr_id,
-                        mfr_model,
-                        mfr_revision,
-                        mfr_date,
-                        ic_device_id,
-                        ic_device_rev,
-                        voltage_sensors: _,
-                        current_sensors: _,
-                    } = self.scratch
-                    else {
-                        unreachable!()
-                    };
-                    dev.read_block(CommandCode::MFR_ID as u8, mfr_id)?;
-                    dev.read_block(CommandCode::MFR_MODEL as u8, mfr_model)?;
-                    dev.read_block(
-                        CommandCode::MFR_REVISION as u8,
-                        mfr_revision,
-                    )?;
-                    dev.read_block(CommandCode::MFR_DATE as u8, mfr_date)?;
-                    dev.read_block(
-                        CommandCode::IC_DEVICE_ID as u8,
-                        ic_device_id,
-                    )?;
-                    dev.read_block(
-                        CommandCode::IC_DEVICE_REV as u8,
-                        ic_device_rev,
-                    )?;
-                    Ok(self.scratch)
-                })
+                self.tx_buf.try_encode_inventory(
```

Comment:
Nit: there's a bunch of places where `dev.component_id().as_bytes()` adds another indentation level to `try_encode_inventory`.  I think getting the name first would prevent this, making the diff smaller and less indented:
```rust
let name = dev.component_id.as_bytes();
self.tx_buf.try_encode_inventory(sequence, name, || {
    // ... etc
```

Reply:
done in https://github.com/oxidecomputer/hubris/pull/2203/commits/a80c20db438b3cf82e47d0a8ca9dc54a9b1d3992!